web/src/components/ResourceCard/ResourceCardSummary.tsx

```diff
@@ -5,35 +5,43 @@ import Date from 'utils/Date';
 import * as S from './ResourceCard.styled';
 
 interface IProps {
-  summary?: TSummary;
+  summary: TSummary;
+  shouldShowResult?: boolean;
 }
 
-const ResourceCardSummary = ({summary}: IProps) => (
+const ResourceCardSummary = ({
+  summary: {
+    lastRun: {time, passes, fails},
+  },
+  shouldShowResult = true,
+}: IProps) => (
   <>
     <div>
       <S.Text>Last run time:</S.Text>
-      <Tooltip title={Date.format(summary?.lastRun?.time ?? '')}>
-        <S.Text>{Date.getTimeAgo(summary?.lastRun?.time ?? '')}</S.Text>
+      <Tooltip title={Date.format(time ?? '')}>
+        <S.Text>{Date.getTimeAgo(time ?? '')}</S.Text>
       </Tooltip>
     </div>
 
-    <div>
-      <S.Text>Last run result:</S.Text>
-      <S.Row>
-        <Tooltip title="Passed assertions">
-          <S.HeaderDetail>
-            <S.HeaderDot $passed />
-            {summary?.lastRun?.passes}
-          </S.HeaderDetail>
-        </Tooltip>
-        <Tooltip title="Failed assertions">
-          <S.HeaderDetail>
-            <S.HeaderDot $passed={false} />
-            {summary?.lastRun?.fails}
-          </S.HeaderDetail>
-        </Tooltip>
-      </S.Row>
-    </div>
+    {shouldShowResult && (
```

Comment:
Something weird is happening with the grid here. Probably because the grid expects one more column.

![Screenshot 2022-11-15 at 11 03 35](https://user-images.githubusercontent.com/3879892/201968333-a871f3dc-e46c-4aba-b420-a459d0f20805.png)